tools/dev-setup.sh

```diff
@@ -103,6 +105,7 @@ config["debug"] = True
 config["driver"] = "${DRIVER}"
 if config["driver"] == "messaging":
     config["messaging_ssl_ca"] = "${CA_PATH}"
+    config["messaging_address"] = socket.gethostname()
```

Comment:
even though this is a config meant for messaging, its not really something that we'd want to setup in the general purpose config when running across multiple nodes, especially given that this configuration change is being reset in the catalogs.